movai_core_shared/messages/metric_data.py

```diff
@@ -0,0 +1,101 @@
+"""
+metrics request fields:
+{
+    'req_type': "metrics",
+    "created": "##########",
+    "robot_info": {
+        "fleet": "movai",
+        "robot_id": "########",
+        "robot": "robotname"
+        "service_name: "servicename"
+    },
+    'req_data': {
+        "metric_type": "alerts",
+        'measurement': "metric_logs",
+        "metric_data": {}
+        }
+    }
+}
```

Comment:
maybe it should be an example inside a read me *or* in the file header 
or should be deleted